### PR TITLE
pkp/pkp-lib#1488 Improvements to workflow

### PR DESCRIPTION
**Pull request by NateWr**
Replaces #1685

### CODE REVIEW

File: templates/workflow/reviewHistory.tpl [hunk 1]
```diff
@@ -9,39 +9,42 @@
...
+		{if $reviewAssignment->getDateReminded() != ''}
+			<div>
+				<strong>{$reviewAssignment->getDateReminded()|date_format:$datetimeFormatShort}</strong>
+				{translate key="common.reminder"}
+			</div>
+		{/if}
+		{if $reviewAssignment->getDateConfirmed() != ''}
+			<div>
+				<strong>{$reviewAssignment->getDateConfirmed()|date_format:$datetimeFormatShort}</strong>
+				{translate key="common.reminder"}
```
Comment (bozana) [inline templates/workflow/reviewHistory.tpl]: The key should be "common.confirm" I think.

File: controllers/modals/editorDecision/form/EditorDecisionWithEmailForm.inc.php [hunk 2]
```diff
@@ -114,31 +114,46 @@ function readInputData() {
 	}
 
 	/**
-	 * @copydoc Form::fetch()
+	 * @see Form::fetch()
```
Comment (bozana) [inline controllers/modals/editorDecision/form/EditorDecisionWithEmailForm.inc.php]: If @see is used than there should be the new parameter definitions/comments...

File: locale/en_US/submission.xml [hunk 3]
```diff
@@ -503,7 +503,7 @@
 	<message key="submission.upload.uploadFiles">Upload Files</message>
 	<message key="submission.upload.response.description">After reviewing the assigned file, record any comments on the file below, and/or upload an annotated copy of the file (or another file).</message>
 	<message key="submission.upload.productionReady">Upload a Production Ready File</message>
-	<message key="submission.upload.proof">Upload a Proof</message>
+	<message key="submission.upload.proof">Upload a File Ready for Publication</message>
```
Comment (bozana) [inline locale/en_US/submission.xml]: When changing the translation for a key, the key should be removed from other locale/languages files, in order for it to be considered for translation again, I think.

File: templates/controllers/wizard/fileUpload/form/fileUploadForm.tpl [hunk 4]
```diff
@@ -193,9 +193,6 @@
 			{include file="controllers/fileUploadContainer.tpl" id="plupload"}
 		{/fbvFormSection}
 
-		{translate|assign:"maxFileUploadStatus" key="common.fileUpload.maxFileSizeStatus" fileSize=$maxFileUploadSize supportName=$currentContext->getSetting('supportName') supportEmail=$currentContext->getSetting('supportEmail')}
```
Comment (bozana) [inline templates/controllers/wizard/fileUpload/form/fileUploadForm.tpl]: Should the variable maxFileUplaodSize then also be removed from the Form (class)?

File: locale/en_US/submission.xml [hunk 5]
```diff
@@ -332,12 +332,11 @@
...
 	<message key="submission.layout.deleteGalleyStylesheet">Delete this style sheet.</message>
 	<message key="submission.layout.editGalley">Edit a Layout Galley</message>
-	<message key="submission.layout.galleyFileData">Galley File Information</message>
 	<message key="submission.layout.galleyFormat">Galley Format</message>
 	<message key="submission.layout.galleyHTMLData">HTML Galley Files</message>
 	<message key="submission.layout.galleyImages">Images</message>
-	<message key="submission.layout.galleyLabelInstructions">The galley label is used to identify the item's file format (e.g. HTML, PDF, etc.).</message>
-	<message key="submission.layout.galleyLabel">Label</message>
+	<message key="submission.layout.galleyLabel">Galley Label</message>
+	<message key="submission.layout.galleyLabelInstructions">Typically used to identify the file format (e.g. PDF, HTML, etc.).</message>
```
Comment (bozana) [inline locale/en_US/submission.xml]: Also here, maybe to remove these keys from the other locales...

File: controllers/modals/editorDecision/form/EditorDecisionWithEmailForm.inc.php (hunk 2 above)
Comment (bozana) [inline controllers/modals/editorDecision/form/EditorDecisionWithEmailForm.inc.php]: Oh, I see you changed it again, sorry!

File: locale/en_US/submission.xml (hunk 5 above)
Comment (NateWr) [inline locale/en_US/submission.xml]: Hmm, I'm not sure about this. The English updates are less-than-critical updates to phrasing for clarity. I wonder if it's best keeping the existing translations for all these languages so close to a release, knowing that they are still essentially accurate and many might not update in time for OJS 3.0.

@asmecher or @mtub, do you have an opinion on this?

Comment (bozana) [inline locale/en_US/submission.xml]: I agree... So lets wait a little bit what others say, else we could merge it like this...

Comment (asmecher) [inline locale/en_US/submission.xml]: I don't see any downside to merging this; other translations based on the older English will be serviceable, and newer translations may also be clearer. Translators won't be notified of the change, thus the workload for them isn't increased.

**Pull request merged**